core/trino-main/src/test/java/io/trino/operator/TestExchangeClient.java

```diff
@@ -586,9 +587,9 @@ public void testTaskFailure()
         URI location3 = URI.create("http://localhost:8080/3");
         URI location4 = URI.create("http://localhost:8080/4");
 
-        processor.addPage(location1, createSerializedPage("location-1-page-1"));
-        processor.addPage(location4, createSerializedPage("location-4-page-1"));
-        processor.addPage(location4, createSerializedPage("location-4-page-2"));
+        processor.addPage(location1, createSerializedPage(1));
+        processor.addPage(location4, createSerializedPage(2));
+        processor.addPage(location4, createSerializedPage(3));
```

Comment:
Why not construct page content anymore?

Reply:
I realized there was already a method for creating pages. I removed the one I added at some point.